### ISSUE TITLE
need enhanced messaging from Breez plug-in for BTCPay

### ISSUE THREAD

**Issue by dme1sc** [Nisaba/btcpayserver-plugins#2]
Hi, what is needed to get Serilog plugin to send Breez plugin fo btcpay to send the breez logs to slack also?
I found that the Breez plugin generates logs but are not inlcuded.
Thanks!  Love the logging to slack - works great.

**Comment by Nisaba** [comment]
Hi,
Yes it seems that the Serilog plugin sends only logs from BTCPay core but not logs from other plugins...

I need to see that and publish an update if it's feasible

**Comment by dme1sc** [comment]
side note - your lightning address is coming up invalid...
donate@btcpay.nisaba.solutions

**Comment by Nisaba** [comment]
yes, you're right ! I have to update this...
You can try donate@btcpay2.nisaba.solutions

**Comment by Nisaba** [comment]
Hi,
I don't use Breeze plugin but I tried with another plugin (B2P Central) and it works...
Are you sure the Minimum information level set in Serilog plugin config is OK with the notification levels from Breeze logs ?

**Comment by dme1sc** [comment]
I don't - how would i check that?

Doug
-- 
Doug Morton | ***@***.*** | 843-936-0632
AlpacaRMS <https://alpacarms.com> - Retail Management System and Point of
Sale

To contact the Helpdesk, please send an email to:
***@***.*** from your main account email on file.
Create a separate email per issue and put a brief summary of the issue in
your subject.  Check case status at:
https://carolinapos.on.spiceworks.com

**Comment by Nisaba** [comment]
here
![image](https://github.com/user-attachments/assets/c0b9c292-f62f-4760-86ca-f2bd28375e1e)

And you compare with the level for the breeze notifications in the log file

**Comment by Nisaba** [comment]
Hi,
Did you resolved your issue ?

**Comment by dme1sc** [comment]
No.  It's a coding thing with the breez plugin.  The developer and I
discussed.  He doesnt have the time to code the serilog levels for the
messaging I want.  I was going to get my dev to take a look but my dev
isn't trained on the code.  It's this sdk:
https://sdk-doc-greenlight.breez.technology/guide/logging.html
I'm basically just wanting to code the Channel Closed messages as Warnings
and then catch them in your plugin to send to Slack.


Doug
--

**Comment by Nisaba** [comment]
Yes, even in BTCPay core, event levels are not always well defined...
Updating log levels is not difficult for the breez developper, only a couple of minutes for that...

But the problem I guess is the publication process againt other updates in their code...
